src/main/java/org/jvnet/hudson/WireMockRuleFactory.java

```diff
@@ -0,0 +1,60 @@
+package org.jvnet.hudson.test;
+
+import com.github.tomakehurst.wiremock.common.SingleRootFileSource;
+import com.github.tomakehurst.wiremock.core.Options;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+/**
+ * Provides a way to record and mock REST calls.  The underlying structure is 
+ * <a href="http://wiremock.org/">WireMock</a>.  To use: 
+ * <pre>
+ * {@code
+ * WireMockRuleFactory wmrf = new WireMockRuleFactory(api_to_mock);
+ * @Rule
+ * WireMockRule = wmrf.getRule(8089); //port is configurable  
+ * }
+ * </pre>
+ *
+ * Point your system to "http://localhost:8089/" to see mocked responses.
+ *
+ * To record mocks: {@code mvn clean test -DrecordMocks=true}  Clearing 
+ * previously recorded mocks before generating new ones is recommended. 
+ */
+public class WireMockRuleFactory {
+    private String prop = System.getProperty("toRecord");
```

Comment:
given that this property is completely under your own control, I would have this at least include wiremock, e.g. `wiremock.record` and I would use the same property name both for the system property as for the pom property